oembed/Oembed.php

```diff
@@ -103,13 +103,14 @@ protected static function matches_scheme($url, $scheme) {
 	 */
 	protected static function autodiscover_from_url($url)
 	{
-		// Fetch the URL (cache for a week by default)
-		$service = new RestfulService($url, 60 * 60 * 24 * 7);
-		$body = $service->request();
-		if (!$body || $body->isError()) {
+		$service = new GuzzleHttp\Client();
+
+		$request = $service->request('GET', $url);
+		// Fetch from Oembed URL (cache for a week by default)
+		if(!$request || $request->getStatusCode() !== 200) {
```

Comment:
Comment says “Fetch from Oembed URL (cache for a week by default)”, but there’s no caching logic here any more :(. We could use `SS_Cache` instead, or just leave out the caching and remove this comment